### PR TITLE
bump dc-tools for datacube-index

### PR DESCRIPTION
**Pull request by jmettes**

### CODE REVIEW

**Comment by emmaai** [comment]
it requires `geos` to install `--no-binary shapely`.

File: index/constraints.txt [hunk 1]
```diff
@@ -1,41 +1,40 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
 #
-#    pip-compile --extra-index-url=https://packages.dea.ga.gov.au/ --output-file=constraints.txt requirements.txt
+#    pip-compile --extra-index-url=https://packages.dea.ga.gov.au/ --output-file=constraints.txt --pre requirements.txt
```
Comment (omad) [inline index/constraints.txt:5]: I don't think you want the `--pre` here, it looks like it accepts pre-release versions of packages.

**Review by omad — CHANGES_REQUESTED** [review]

File: index/constraints.txt (hunk 1 above)
Comment (jmettes) [reply index/constraints.txt:5]: I was following instructions in the README. I update it to not use `--pre` for releases?
https://github.com/opendatacube/datacube-docker/tree/main/index#how-to-create-a-new-image-with-latest-odc-tools

**Comment by jmettes** [comment]
pre-commit.ci seems to have been activated, even though there is no `.pre-commit-config.yaml` file for this repo.

image

Maybe someone with access can remove it from the pre-commit.ci managed repos? I don't have access to.
https://results.pre-commit.ci/install/github/30752170

**Review by emmaai — APPROVED** [review]
lgtm

**Review by omad — APPROVED** [review]

**Pull request merged**